codespell_lib/data/dictionary.txt

```diff
@@ -46828,6 +46828,8 @@ pronouced->pronounced
 pronounched->pronounced
 pronounciation->pronunciation
 pronounciations->pronunciations
+pronte->prone, pronate, pronto,
+pronted->printed, pronated, pronged,
```

Comment:
Perhaps add otherl verbal forms?
```suggestion
pront->print, pronto, prone, front, prong,
pronte->prone, pronate, pronto, ponte,
pronted->printed, pronated, pronged, proned,
prontes->pronates, prones,
pronting->printing, pronating, proning,
pronts->prints, prongs, pronto,
```

Reply:
Yes, absolutely. Thanks a lot for the suggestion.

Unfortunately GitHub seems to be buggy currently because when trying to apply the suggestion here it just showed "The diff has changed, please refresh" or similar and haven't allowed me to apply it.

Thus had force pushed a new version of this commit via command line, this did the trick as well.